piety-promoted-v2/modernized/03-chapter-02.adoc

```diff
@@ -330,17 +335,21 @@ and returned, he pursuing his journey to the Land`'s-End; from which he returned
 visiting Friends through the counties till he got home,
 which was on the 30th of the Sixth month.`'
 
-I had a very satisfactory journey, '`says he, '`for the Lord was with me,
+"`I had a very satisfactory journey,`' says he, '`for the Lord was with me,
 and was to me mouth and wisdom, language and utterance;
-blessed be his glorious name forever.
+blessed be his glorious name forever.`'
+
+[.embedded-content-document.letter]
```

Comment:
I see why you embedded this... because it has a signature and SSCC at the bottom. But... it seems kinda weird to me to have the quotation start in the paragraph above, and then have it continue in the embedded section. Since this doesn't seem to be a letter or written testimony, etc. I think we should not embed it, but still go ahead and keep the signature and SSCC at the end. Hope that makes senes!